cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/WidgetCollection/index.tsx

```diff
@@ -161,7 +146,7 @@ const WidgetCollectionView: React.FC<IWidgetCollectionProps> = ({
       <div className={classes.widgetContainer}>
         {activeWidgets.map((widgetID, widgetIndex) => {
           return (
-            <If condition={widgetInfo[widgetID]} key={widgetIndex}>
+            <If key={widgetIndex} condition={widgetInfo.get(widgetID) !== undefined}>
```

Comment:
Instead of checking if widgetInfo.get(widgetID) is undefined, you can use has() to check if the key exists

Reply:
Resolved